less/v2/inventory.less

```diff
@@ -183,6 +183,11 @@
         box-shadow: none;
         filter: drop-shadow(0 0 1px var(--dnd5e-color-black));
 
+        &.tmp.filled { 
+          --icon-fill: var(--dnd5e-color-blue);
+          --icon-stroke: var(--dnd5e-color-blue)
+        }
+        &.tmp:hover { filter: drop-shadow(0 0 2px var(--dnd5e-color-blue)); }
```

Comment:
Slight preference for using purple here. That's loosely what the design uses as the 'magic' colour, like in the Spellcasting header. So either `--dnd5e-color-sc-1` or `--dnd5e-color-sc-2` (probably `--dnd5e-color-sc-2`).

Reply:
![image](https://github.com/user-attachments/assets/c45382c0-1fe6-49be-b7b1-660866bebd20)
Yep, looks good